app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java

```diff
@@ -829,15 +833,33 @@ private void customizePictureViewer(){
         ((IconicsImageView) dialogLayout.findViewById(R.id.ll_apply_theme_3thAct_icon)).setColor(getIconColor());
 
         swApplyTheme_Viewer.setChecked(isApplyThemeOnImgAct());
+        if(isApplyThemeOnImgAct())
+        {
```

Comment:
@angmas1 move the opening bracket of the if next to the starting of the if block. Please follow the existing coding style of the app.